pkg/controller/appdefinition/deploy.go

```diff
@@ -629,7 +629,7 @@ func containerAnnotation(container v1.Container) string {
 }
 
 func resolvedOfferingsAnnotation(appInstance *v1.AppInstance, container v1.Container) (string, error) {
-	if resolved, exists := appInstance.Status.ResolvedOfferings.Containers[container.Name]; exists {
+	if resolved, exists := appInstance.Status.ResolvedOfferings.Containers[strings.ToLower(container.Name)]; exists {
```

Comment:
While stepping through the debugger, I found a case here where `container.Name` was capitalized for some reason, which is why I added this `strings.ToLower`.

Reply:
Interesting, I thought container name is not allowed to have capital name